tools/tesseratos/src/tesseratos/debug_camera/plugin.cpp

```diff
@@ -63,9 +65,10 @@ void tesseratos::debugCameraPlugin(Cubos& cubos)
     cubos.startupSystem("create Debug Camera").call([](Commands commands, State& debugCamera) {
         debugCamera.entity = commands.create()
                                  .named("debug-camera")
-                                 .add(PerspectiveCamera{
-                                     .active = false,
+                                 .add(Camera{
+                                    .active = false // this needs to be done because active was moved from PerspectiveCamera to Camera
```

Comment:
You can remove the comment - future readers won't know that you changed this :relieved:
In these cases when your goal is to make the reviewer's job easier, you can add a comment yourself to your own PR.